### ISSUE TITLE
Unexpected Kanban Interaction with Linked Shelves

### ISSUE THREAD

**Issue by Djhyv2** [ubergarm/RimWorld-KanbanStockpile#28]
When using linked storages, the settings of Kanban stockpile seem to apply to unrelated groups.

I tested the following on 1.4 using only Harmony, Core, Biotech, Ideology, Royalty, and Kanban Stockpile.

Created 4 small shelves and performed the following tests.

Linked no shelves, updated Kanban settings on single shelf, only single shelf had updated settings
Linked 2 shelves, updated Kanban settings on an unlinked shelf, only single shelf had updated settings
Linked 2 shelves, updated Kanban settings on a linked shelf, both linked shelves had updated settings
Linked 2 distinct groups of 2 shelves, updated Kanban settings on a linked shelf, both sets of linked shelves had updated settings.

**Comment by ubergarm** [comment]
Right, I need to update the description of the mod. I mentioned in a comment long ago that is now buried on the second or third page hah.

The mod was made before linked storage was a thing, and does not work with linked storage. It gives the strange behavior folks describe like some common shared settings that isn't real.
